### DISCUSSION TITLE
Does SQLAlchemy support in Python2?

### DISCUSSION THREAD

**Discussion by DavidShilol** [trinodb/trino-python-client#460] [Q&A] [answered]
Hi all,
    
I execute `pip install 'trino[sqlalchemy]'` in Python2, but i occurred error that "WARNING: trino 0.305.0 does not provide the extra 'sqlalchemy'".
    
Is there any way that I can use trino-orm in Python2?

**Comment by hashhar** [comment, marked as answer]
The latest trino-python-client that supports Python 2 was 0.305.0.
The SQLAlchemy support was added in 0.307.0.

So there's no version of trino-python-client that supports SQLAlchemy AND Python2.

The docs also clearly mention only Python >= 3.8 is supported.

**Reply by DavidShilol** [reply]
Thanks for answering my question! Now I can entirely give up.
